test/composites/menu/events/main.js

```diff
@@ -72,9 +73,22 @@ describe('composites/menu/events/main', () => {
       assert.equal(dropdown.getAttribute('aria-expanded'), 'false');
       fire(elements.topBar.querySelector('#drop-1'), 'click');
       setTimeout(() => {
-      assert.equal(dropdown.getAttribute('aria-expanded'), 'true');
+        assert.equal(dropdown.getAttribute('aria-expanded'), 'true');
         done();
-      }, 400); // give animation/other timeouts a change to do stuff
+      }, 400); // give animation/other timeouts a chance to do stuff
+    });
+
+    it('clicking closed dropdown should open that dropdown and close any other open dropdowns', (done) => {
+      const dropdown = elements.dropdown;
+      const dropdown2 = elements.dropdown2;
+      // dropdown.setAttribute('aria-expanded', 'true');
+      fire(dropdown, 'click');
```

Comment:
I think I see what you were going for here but shouldn't we draw it out and make an assertion before the 2nd click (in the `setTimeout`)?

```js
assert.equal(dropdown2.getAttribute('aria-expanded'), 'true');
```

Reply:
Very good point.